docs/examples.md

```diff
@@ -136,6 +136,18 @@ To learn more about the Clojure representation of a node please see our [represe
                                    :last-name "Anderson"}})
 ~~~
 
+### Find a single node in Neo4j
+
+The entry given to `find-node!` is a Node Lookup representation. To learn more about the Clojure Node Lookup representation see our [representations](representations.md) page
```

Comment:
Now the three first paragraphs repeat (see below):
> ... To learn more about the Clojure Node Lookup representation see ...

Is there a better way to turn attention to the representations without repeating it?